packages/building-rollup/test-node/integration.test.js

```diff
@@ -9,7 +8,13 @@ const { startDevServer } = require('@web/dev-server');
 
 const rootDir = path.resolve(__dirname, '..', 'dist');
 
-describe('integration tests', () => {
+describe('integration tests', async () => {
+  let expect;
+  before(async () => {
+    const chai = await import('chai');
```

Comment:
`({expect}=async import(‘chai’));`?

Reply:
Happy to do this if you have a strong opinion but if it's ok I'll leave this for now. There's once place at least where we need to call `chai.use(..)` in between anyways and I think we can stop doing this in a `before` once we are using ESM anyways.

Reply:
Separately, does `@open-wc/testing` do this for consumers, already? Or will this be a requirement to migrating?

Reply:
Not a blocker. Mainly pointing it out in the case end users need to do this every time.

Reply:
Yeah we do this for consumers. We load up all the plugins and then return a instance of chai with those loaded. Found a issue with chai while checking that out that we gotta fix before landing this: https://github.com/open-wc/open-wc/pull/2795/files#r1599496849